[2022-07-04 23:33:04,189] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: Teste_ETL_Raízen.xls_to_xlsx scheduled__2022-07-04T22:30:00+00:00 [queued]>
[2022-07-04 23:33:04,262] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: Teste_ETL_Raízen.xls_to_xlsx scheduled__2022-07-04T22:30:00+00:00 [queued]>
[2022-07-04 23:33:04,262] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-07-04 23:33:04,263] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2022-07-04 23:33:04,263] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-07-04 23:33:04,400] {taskinstance.py:1377} INFO - Executing <Task(PythonOperator): xls_to_xlsx> on 2022-07-04 22:30:00+00:00
[2022-07-04 23:33:04,456] {standard_task_runner.py:52} INFO - Started process 1082 to run task
[2022-07-04 23:33:04,499] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'Teste_ETL_Raízen', 'xls_to_xlsx', 'scheduled__2022-07-04T22:30:00+00:00', '--job-id', '45', '--raw', '--subdir', 'DAGS_FOLDER/ETL_Raizen.py', '--cfg-path', '/tmp/tmp09k639qw', '--error-file', '/tmp/tmpc537kpya']
[2022-07-04 23:33:04,504] {standard_task_runner.py:80} INFO - Job 45: Subtask xls_to_xlsx
[2022-07-04 23:33:04,931] {task_command.py:369} INFO - Running <TaskInstance: Teste_ETL_Raízen.xls_to_xlsx scheduled__2022-07-04T22:30:00+00:00 [running]> on host d342cad78eb0
[2022-07-04 23:33:05,750] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=Teste_ETL_Raízen
AIRFLOW_CTX_TASK_ID=xls_to_xlsx
AIRFLOW_CTX_EXECUTION_DATE=2022-07-04T22:30:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-07-04T22:30:00+00:00
[2022-07-04 23:33:55,252] {python.py:173} INFO - Done. Returned value was: None
[2022-07-04 23:33:55,459] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=Teste_ETL_Raízen, task_id=xls_to_xlsx, execution_date=20220704T223000, start_date=20220704T233304, end_date=20220704T233355
[2022-07-04 23:33:55,739] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-07-04 23:33:56,453] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
